spark/datadog_checks/spark/data/conf.yaml.example

```diff
@@ -109,35 +109,6 @@ instances:
     #
     # streaming_metrics: true
 
-    ## @param tags - list of strings - optional
```

Comment:
is it worth overriding `tags` priority to be higher up?

Reply:
likely not

Reply:
lgtm